components/support/test/src/main/java/mozilla/components/support/test/Assert.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+@file:JvmName("AssertUtils")
+
+package mozilla.components.support.test
+
+/**
+ * Throw assertion error.
+ *
+ * @param message optional message for error
+ */
+@JvmOverloads
+fun fail(message: String? = null): Nothing {
+    message ?: throw AssertionError()
```

Comment:
I'd prefer to use either an `if` statement or `.let`, so that the `AssertionError` with a message is thrown next to the not-null check.
```suggestion
    message?.let { throw AssertionError(it) } ?: throw AssertionError()
```
```suggestion
    throw if (message != null) AssertionError(message) else AssertionError()
```

Reply:
Okay, I'll use `if-else` statement here.